packages/forma-36-react-components/src/components/Switch/style.css

```diff
@@ -0,0 +1,55 @@
+@import 'settings/colors';
+
+.Switch__wrapper {
+  display: flex;
+}
+
+.Switch {
+  width: 35px;
+  height: 21px;
+  background: var(--color-element-mid);
+  border-radius: 21px;
+  cursor: pointer;
+  position: relative;
+  outline: none;
+  margin-right: 0.5rem;
+  border: 0;
+}
+
+.Switch:before {
+  position: absolute;
+  content: '';
+  width: 17px;
+  height: 17px;
+  left: 2px;
+  top: 2px;
+  background: var(--color-white);
+  -webkit-transition: 0.1s;
```

Comment:
@azer +1 for tokens. Also you can use unprefixed css properties here, so just transition. I'd also recommend specifying which properties you want to target for the transition.

```css
transition: left var(--transition-easing-default) var(--transition-duration-default);
```

In general I think there's a better approach for styling this component. Here's an example: https://codepen.io/anon/pen/mgOeyL. This uses border for the spacing between the inner and outer components, which means you can easy to understand values for the transition - which should use `transform` rather than `left`/`right` for performance purposes